### ISSUE TITLE
add support for monitors with frequencies other than 60Hz

### ISSUE THREAD

**Issue by pfedick** [pfedick/DeckerGame#91]

**Comment by pfedick** [comment]
On my FreeBSD  I get only 30 FPS, and everything is quite slow. Especially walking and physics. Both is connected to the framerate.  But the game runs on FreeBSD :-)  And I can use this machine for testing.

I should change this mechanism and use time instead.
I could also disable synchronisation to Monitor Hz or force framerate to a specific number (or both).

**Comment by pfedick** [comment]
When I implemented the particle system, I introduced a frame rate compensation mechanism. When running on 60Hz, which is my development environment, a new frame starts every 16.66667 ms. On faster Framerates this time will be shorter, on slower Framerates this will be longer.

I decided that 60Hz is my baseline, my "1". I calculate a frame rate compensation value every frame and simply multiply every movement by this value. On 60 Hz systems this value will always be near 1.0. On faster systems, 90 Hz for example, this value will be lower, so that movement per frame gets slower, on slower systems the value will get higher.
